client/src/boot/index.js

```diff
@@ -53,7 +53,6 @@ async function appBoot() {
     routes.setStore(store);
     routes.start(window.location.pathname);
 
-    // @todo - Remove once we remove entwine
```

Comment:
We probably should create common ticket to clean up JS files after we remove entwine.

Reply:
We can worry about that when/if that ever happens